scripts/upgrade_admin_server.sh

```diff
@@ -116,6 +116,13 @@ n.save"
     rm -f /etc/systemd/system/crowbar.service
     systemctl daemon-reload
 
+    # Make sure latest chef data are uploaded
+    # Normally packages should take care of it, but for the rare case of chef-server
+    # not being available during the upgrade, better make sure and do it explicitely.
+    systemctl restart chef-server
+    pushd /opt/dell/crowbar_framework
+    sudo -u crowbar RAILS_ENV=production bin/rake chef:upload:all
```

Comment:
Hm, do you think we should wait until it is started?

Just a reminder - this is already a backup solution, because normally chef data are uploaded already during package installation and chef-server should not be stopped at that time.